bin/img-set.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
```

Comment:
Question (non-blocking): Is there any reason not to rename this as a `.js` file and run it as a node script?

Without this, thing like Prettier and ESLint likely won't keep check of it.

Reply:
Probably not. This is just a legacy of starting the process in bin 🤦‍♀️